isis/tests/Fixtures.cpp

```diff
@@ -255,6 +255,48 @@ namespace Isis {
   }
   
 
+  void ObservationPair::SetUp() {
+      FileName labelPathL = FileName("data/stereoPair/stereoImageL.pvl");
+      FileName labelPathR = FileName("data/stereoPair/stereoImageR.pvl");
+
+      isdPathL = new FileName("data/stereoPair/stereoImageL.isd");
+      isdPathR = new FileName("data/stereoPair/stereoImageR.isd");
+
+      cubeL = new Cube();
+      cubeR = new Cube();
+
+      cubeLPath = tempDir.path() + "/stereoPairL.cub";
+      cubeRPath = tempDir.path() + "/stereoPairR.cub"; 
+
+      cubeL->fromIsd(cubeLPath, labelPathL, *isdPathL, "rw");    
+      cubeR->fromIsd(cubeRPath, labelPathR, *isdPathR, "rw");    
+
+      cubeList = new FileList();
+      cubeList->append(cubeL->fileName());
+      cubeList->append(cubeR->fileName());
+
+      cubeListFile = tempDir.path() + "/cubes.lis";
+      cubeList->write(cubeListFile);
+
+      cnetPath = "data/stereoPair/stereoPair.net";
+      network = new ControlNet();
+      network->ReadControl(cnetPath);
+  }
+
+
+  void ObservationPair::TearDown() {
+    delete cubeList;
+    delete network;
+
+    delete cubeL;
+    delete cubeR;
+
+    delete isdPathL;
+    delete isdPathR;
```

Comment:
Wrap these in if statements in case there is an error during setup and some of these are still null.